.github/workflows/test.yml

```
@@ -158,7 +161,7 @@ jobs:
       docker-image-override: agglayer_image
       docker-tag: ${{ needs.docker-build-local.outputs.tags }}
       kurtosis-cdk-ref: 2527f38a96cedba1bff6cc94848ae77de8d44a37 # jihwan/integrations
-      docker-artifact-name: docker-image
+      docker-artifact-name: ${{ github.event_name == '"workflow_dispatch"' && 'docker-iamge' || '' }}
```

```suggestion
      docker-artifact-name: ${{ github.event_name == 'workflow_dispatch' && 'docker-image' || '' }}
```
This looks weird to me: why would we not override the `agglayer_image` when running from push?

---

> This looks weird to me: why would we not override the `agglayer_image` when running from push?

When running from push, the image gets built and pushed to ghcr and subsequently picked up from there. When running form merge queue, it gets picked from artifacts.

This has to match what the docker build job does. We now have a number of conditionals that have to be consistent. This is one more such conditional. That's what I meant when I pointed out in the other comment that the code is becoming spaghetti-esque.

---

ofc I got the condition the other way around :facepalm: